### ISSUE TITLE
migrate content from dominode.dm to staging server

### ISSUE THREAD

**Issue by gubuntu** [dominodeorg/dominode#23]
- back up dominode.dm (old instance)
- restore to staging GeoNode (only layers that are not in the staging server, which came from dominode.piensa.co)

**Comment by lucernae** [comment]
@gubuntu since Peter provisions the new prod server, what do we do about them at the moment?
Do we just treat it as two staging instance? (one is @ingenieroariel 's machine and the other is the one Peter provisions) ?

**Comment by gubuntu** [comment]
we're not doing anything on Peter's node just yet other than retrieving dominode.dm. 

That needs to be restored to the staging PG / GeoServer / GeoNode you've set up on Ariel's node. I hope you noted that I wanted to keep what was on the staging geonode up till now since that has styles and metadata we developed during training, that I wanted to review and migrate if necessary. But don't let that be a blocker or timesink. 

Once we and the GoCD officials have been using the staging server a bit and the DVRP have approved our report then we proceed with deploying to Peter's node - we can discuss how we deal with that leading up to July when it must be full production.
